.travis.yml

```diff
@@ -15,18 +15,18 @@ env:
 
 matrix:
   include:
-    - python: "2.7"
-      dist: trusty
-      env: DJANGO=1.11 COVERAGE="coverage run -m" TEST_K="not ddns_client" TEST_OPTS="--pep8"
```

Comment:
guess the coverage and pep8 needs to be done somewhere else when killing it on py27.